lib/webrick-patch.rb

```diff
@@ -14,7 +14,7 @@ def setup_ssl_context(config) # :nodoc:
       end
       ctx = OpenSSL::SSL::SSLContext.new
       ctx.set_params
-      ctx.ciphers = CIPHERS
+      ctx.ciphers = (CIPHERS.split(':') - config.fetch(:SSLDisabledCiphers, '').split(':')).join(':')
```

Comment:
Have you committed all the changes? I don't see how values from ```ssl_disabled_ciphers``` get into ```SSLDisabledCiphers```. I would like to avoid introducing custom webrick config options however; you can access global settings directly from here. Additionally, you can set a default value for ```ssl_disabled_ciphers``` to an empty array, which will allow you to dispense with defensive code around fetching it.